pwndbg/disasm/__init__.py

```diff
@@ -85,7 +85,7 @@ def get_disassembler(pc):
     if pwndbg.arch.current in ('arm', 'aarch64'):
         extra = {0:CS_MODE_ARM,
                  0x20:CS_MODE_THUMB}[pwndbg.regs.cpsr & 0x20]
-
+    pwndbg.arch.mode={CS_MODE_ARM:" ARM MODE",CS_MODE_THUMB:" THUMB MODE",None:""}[extra]
```

Comment:
'MODE' string is already added 
{CS_MODE_ARM:" ARM MODE",CS_MODE_THUMB:" THUMB MODE",None:""}
Putting it besides banner will make it print always. 
But on x86 this would not make any sense.
E.g on x86:
Banner would look like DISASM MODE
On ARM:
Banner would look like DISASM THUMB MODE